syncr_backend/util/drop_util.py

```diff
@@ -125,50 +125,64 @@ class PermissionError(Exception):
     :param file_name: name of file to be added/removed.
 
     """
-    peers = await get_drop_peers(drop_id)
-    old_drop_metadata = await get_drop_metadata(drop_id, peers)
+    drop_directory = await get_drop_location(drop_id)
+    old_drop_m = await DropMetadata.read_file(
+        id=drop_id,
+        metadata_location=os.path.join(
+            drop_directory, DEFAULT_DROP_METADATA_LOCATION,
+        ),
+    )
+
+    if old_drop_m is None:
```

Comment:
It first tries to read the file, then if that fails tries to get it from the network.  I don't want to call get_drop_metadata right at the beginning since I don't want to have to look up the peers unless I know they're needed